spotbugs/src/xsl/default.xsl

```diff
@@ -50,8 +50,8 @@
 	method="xml"
 	indent="yes"
 	omit-xml-declaration="yes"
-	standalone="yes"
-    doctype-system="http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"
+	standalone="omit"
```

Comment:
This is the only actual change. The rest is spaces to tabs to align with the rest of the majority of the file.